modules/015-admission-policy-engine/docs/README.md

```diff
@@ -6,20 +6,36 @@ description: Deckhouse admission-policy-engine module enforces the security poli
 This module enforces the security policies in the cluster according to the Kubernetes [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) using the [Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/docs/) solution.
 
 The Pod Security Standards define three different policies to broadly cover the security spectrum. These policies are cumulative and range from highly-permissive to highly-restrictive:
-- `Privileged` — Unrestricted policy. Provides the widest possible permission level (used by default).
-- `Baseline` — Minimally restrictive policy which prevents known privilege escalations. Allows for the default (minimally specified) Pod configuration.
+- `Privileged` — Unrestricted policy. Provides the widest possible permission level;
+- `Baseline` — Minimally restrictive policy which prevents known privilege escalations. Allows for the default (minimally specified) Pod configuration;
 - `Restricted` — Heavily restricted policy. Follows the most current Pod hardening best practices.
```

Comment:
Shouldn't it be semicolon at the end here? According to previous paragraphs.

Reply:
Maybe I'm wrong, but there are already semicolons there in actual version.

Reply:
NVM